asteroid/masknn/blocks.py

```diff
@@ -88,6 +88,7 @@ def __init__(self, in_chan, n_src, out_chan=None, n_blocks=8, n_repeats=3,
 
         layer_norm = norms.get(norm_type)(in_chan)
         bottleneck_conv = nn.Conv1d(in_chan, bn_chan, 1)
+        self.bottleneck = nn.Sequential(layer_norm, bottleneck_conv)
```

Comment:
I moved the self.bottleneck from the end here as it makes more sense to have it here.

Reply:
That's clearer this way, you're right. Thank you